samples/project-anfield/requirement.txt

```diff
@@ -0,0 +1,2 @@
+cohesity_management_sdk==1.8.1
+cryptography==2.1.4
```

Comment:
Is there a conditional requirement? cryptography is a requirement only if they want to do encryption correct ?

Reply:
Commented and added a note in the file